domains/gb.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "your-github-username",
+        "email": "your-email-address@example.com"
```

replace with your username and email here
```suggestion
        "username": "gbijwe",
        "email": "gaurav.a.bijwe@gmail.com"
```